pkgs/development/python-modules/plexapi/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests, tqdm, websocket_client, pytest, pillow }:
+
+buildPythonPackage rec {
+  pname = "PlexAPI";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "pkkid";
+    repo = "python-plexapi";
+    rev = version;
+    sha256 = "1rzy018zcsws56mcghnphhzwj650pwj7qg6nh9z1kjvgwwjfmghf";
+  };
+
+  propagatedBuildInputs = [ requests tqdm websocket_client ];
+
+  doCheck = true;
+  checkInputs = [ pytest pillow ];
```

or disable python2.7. I'm fine with either tbh
```suggestion
  checkInputs = [ pytest pillow ]
    ++ lib.optionals isPy27 [ mock ];
```

```
builder for '/nix/store/babalivzngycash5y3vk2xra333q9cv1-python2.7-PlexAPI-3.2.0.drv' failed with exit code 1; last 10 log lines:
      from mock import patch, MagicMock
  ImportError: No module named mock


  ----------------------------------------------------------------------
  Ran 7 tests in 0.000s

  FAILED (errors=7)
  Test failed: <unittest.runner.TextTestResult run=7 errors=7 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=7 errors=7 failures=0>
```